src/Tools/_framework/Paths/PortfolioActivityEditor.jsx

```diff
@@ -1689,23 +1712,156 @@ export function PortfolioActivityEditor() {
+                                            />
+                                            Line #
+                                            {warningObj.doenetMLrange.lineBegin}{" "}
+                                            {warningObj.message}
+                                          </ListItem>
+                                        );
+                                      },
+                                    )}
+                                  </List>
+                                </PopoverBody>
+                              </PopoverContent>
+                            )}
+                          </Popover>
+
+                          <Popover offset={[119, 5]}>
+                            <PopoverTrigger>
+                              <Tag
+                                data-test="Error Button"
+                                tabIndex="0"
+                                cursor="pointer"
+                                size="md"
+                                colorScheme={
+                                  errorsAndWarnings.errors.length == 0
+                                    ? "blackAlpha"
+                                    : "red"
+                                }
+                              >
+                                {errorsAndWarnings.errors.length} Error
+                                {errorsAndWarnings.errors.length != 1 && "s"}
+                              </Tag>
+                            </PopoverTrigger>
+                            {errorsAndWarnings.errors.length == 0 ? (
+                              <PopoverContent data-test="Error Content">
+                                <PopoverArrow />
+                                <PopoverHeader fontWeight="semibold">
+                                  No Errors
+                                </PopoverHeader>
+                              </PopoverContent>
+                            ) : (
+                              <PopoverContent data-test="Error Content">
+                                <PopoverArrow />
+                                <PopoverHeader fontWeight="semibold">
+                                  Error
+                                  {errorsAndWarnings.errors.length != 1 && "s"}
+                                </PopoverHeader>
+                                <PopoverBody maxH="40vh" overflow="scroll">
+                                  <List spacing={2}>
+                                    {errorsAndWarnings.errors.map(
+                                      (errorObj, i) => {
+                                        return (
+                                          <ListItem
+                                            key={i}
+                                            data-test={`Error ${i}`}
+                                          >
+                                            <ListIcon
+                                              as={MdCheckCircle}
```

Comment:
Is there an x circle rather than a check circle? A checkmark seems too much like a good thing!

Reply:
WarningTwoIcon - same as the update button